.github/workflows/extract-stealth.yml

```diff
@@ -0,0 +1,30 @@
+name: Extract stealth.min.js
+
+on:
+  push:
+    branches:
+      - master
+    paths:
+      - 'packages/puppeteer-extra-plugin-stealth/**' 
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Extract stealth.min.js
+        run: |
+          cd packages/extract-stealth-evasions
+          npm install
+          node index.js
+
+      - name: Commit stealth.min.js
+        uses: EndBug/add-and-commit@v4
+        with:
+          add: 'packages/extract-stealth-evasions/stealth.min.js'
+          force: true
```

Comment:
As commented, it might be better to push to a separate branch (by adding `ref: 'stealth-js'` for example).

Reply:
Yeah, I think using a dedicated branch for this is best as it gives us more flexibility later on (also reduces noise on `master`). :-)

Reply:
If we're lazy we can just https://gitcdn.xyz/ for the CDN part. We could also make a small snippet to load the stuff into devtools based on the CDN url (https://stackoverflow.com/questions/7474354/include-jquery-in-the-javascript-console)